tests/Feature/Http/Controllers/CharacterControllerTest.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace Tests\Feature\Http\Controllers;
+
+use App\Models\Character;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use JMac\Testing\Traits\AdditionalAssertions;
+use Tests\TestCase;
+
+/**
+ * @see \App\Http\Controllers\CharacterController
+ */
+class CharacterControllerTest extends TestCase
+{
+    use AdditionalAssertions;
+    use RefreshDatabase;
+    use WithFaker;
+
+    /**
+     * @test
```

Comment:
I think these should eventually have better descriptions.

Reply:
I think we'll do a deeper documentation pass at some point, just the Doc annotation is sufficient for now